Documentation/Getting-Started/resource-cleanup.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Ceph Cleanup
+---
+
+## Cleanup up of ceph resources
+
+To cleanup a specific ceph resource, add `ceph.rook.io/force-deletion="true"` annotation to that custom resource before deleting it. Rook will start a cleanup job that will delete the all the ceph resources created by that custom resource. For example, run the following commands to clean the CephFSSubVolumeGroup resource named `myfs-csi`:
+
+``` console
+kubectl annotate cephfilesystemsubvolumegroups.ceph.rook.io myfs-csi -n rook-ceph ceph.rook.io/force-deletion="true"
+kubectl delete cephfilesystemsubvolumegroups.ceph.rook.io myfs-csi -n rook-ceph
+```
+
+Once the cleanup job is completed successfully, Rook will remove the finalizers from the deleted custom resource.
+
+This cleanup is supported only for the following custom resources:
```

Comment:
Since we're going to have multiple types of resources supported, it would be nice to have a table, one row for each resource, and its corresponding resources that will be cleaned up.

Reply:
using table.